packages/graphql/tests/tck/tck-test-files/issues/630.test.ts

```diff
@@ -75,7 +75,7 @@ describe("Cypher directive", () => {
         expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
             "MATCH (this:Actor)
             RETURN this { movies: [this_movies IN apoc.cypher.runFirstColumn(\\"MATCH (m:Movie {title: $title})
-            RETURN m\\", {this: this, auth: $auth, title: $this_movies_title}, true) | this_movies { actorsConnection: apoc.cypher.runFirstColumn(\\"CALL {
```

Comment:
The only thing to note is that when we merge from the hotfix branch to `dev`, this test will be failing because it will expect this removed Cypher. Nothing we can do about it, it's just a heads up.